tools/esdlcomp/esdlcomp.h

```diff
@@ -1087,6 +1087,13 @@ class EspMethodInfo
             tags = p->next;
             delete p;
         }
+
+        if (name_)
+           free(name_);
+        if (request_)
+           free(request_);
+        if (response_)
+           free(response_);
```

Comment:
Not sure if it matters or not, but the constructor should use an initialization list for name_, request_ and response_ to ensure their values are null just in case the strdups in the constructor fail.

Same goes for other classes.